spark3-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestMergeIntoTable.java

```diff
@@ -355,9 +439,17 @@ private void initTable(String tabName) {
     });
   }
 
-  protected void append(String tabName, Employee... employees) throws NoSuchTableException {
-    List<Employee> input = Arrays.asList(employees);
-    Dataset<Row> inputDF = spark.createDataFrame(input, Employee.class);
-    inputDF.coalesce(1).writeTo(tabName).append();
+  private void setWriteMode(String tabName, String mode) {
```

Comment:
I think this should be `setDistributionMode` instead because `setWriteMode` sounds more general, like "copy-on-write" or "merge-on-read".